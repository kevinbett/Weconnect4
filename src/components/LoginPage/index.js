import React from 'react';
import Navigation from '../Navigation/index';
import LoginForm from '../LoginForm/LoginForm'

const LoginPage = () => (

   <div>
       <Navigation />
       < LoginForm/>
      
   </div>
);

export default LoginPage;