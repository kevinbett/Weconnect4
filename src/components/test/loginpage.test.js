import LoginPage from "../LoginPage"
import { shallow } from "enzyme"

describe('Login page',() => { 
    it ('renders Login page', () => { 
        expect(shallow(<LoginPage/>)).toMatchSnapshot()
    });
} )