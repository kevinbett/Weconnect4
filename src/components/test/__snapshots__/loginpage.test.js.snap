// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login page renders Login page 1`] = `
<div>
  <Navigation />
  <LoginForm />
</div>
`;
